packages/belos/src/BelosGmresPolyOp.hpp

```diff
@@ -783,7 +784,7 @@ namespace Belos {
         for(int j=0; j< extra[i]; ++j){
           theta_(count,0) = theta_(i,0);
           theta_(count,1) = theta_(i,1);
-          thetaPert(count,0) = theta_(i,0)+(j+MCT::one())*5e-8;
+          thetaPert(count,0) = theta_(i,0)+(j+MCT::one())*MagnitudeType(5e-8);
```

Comment:
@hkthorn what's this 5e-8 all about?

Reply:
It's a slightly hackish way to get a pseudo-Modified Leja Ordering when the list has repeated values- perturb the repeats by 5e-8, push it through the ordering function, and then use that ordering to reorder the unperturbed values.  There are probably better ways to do it.